.github/workflows/postsubmit.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         with:
           #channel: stable
           cache: true
-          flutter-version: '3.7.12'
+          flutter-version: '3.10.4'
```

Comment:
```
channel: stable
cache: true
#flutter-version: '3.10.4'
```

Reply:
Ok I will change that.
I've seen the comment of @M123-dev in the other repo, but it was too late

Reply:
That should be OK now